code/modules/mob/living/simple_animal/hostile/megafauna/hierophant.dm

```diff
@@ -430,11 +448,16 @@ Difficulty: Hard
 	var/targets_the_same = (new_target == target)
 	. = ..()
 	if(. && target && !targets_the_same)
+		abilities_landed = 0 // Reset for new targets
 		visible_message("<span class='hierophant_warning'>\"[pick(target_phrases)]\"</span>")
 		if(spawned_beacon && loc == spawned_beacon.loc && did_reset)
 			arena_trap(src)
 
 /mob/living/simple_animal/hostile/megafauna/hierophant/adjustHealth(amount, updating_health = TRUE)
+	if(immune)
+		visible_message("<span class='danger'>[src] deflects the attack; [p_they()] can't be hit while focused!</span>", "<span class='userdanger'>You deflect the attack!</span>")
```

Comment:
`;` in a sentence? Probably grammatically correct but I rarely see it in-game so it might stand out as odd.